ang/civicase/case/actions/directives/case-actions.directive.js

```diff
@@ -310,11 +310,19 @@
           },
 
           gotoWebform: function (selectedCase, path, clientId) {
-            clientId = 'cid' + clientId;
-            var url = CRM.url(path, {
-              case1: selectedCase.id,
-              [clientId]: selectedCase.client[0].contact_id
-            });
+            var url;
+            if (clientId) {
+              clientId = 'cid' + clientId;
+              url = CRM.url(path, {
+                case1: selectedCase.id,
+                [clientId]: selectedCase.client[0].contact_id
+              });
+            }
+            else {
```

Comment:
There are linting errors in this file, did you run `npm i` in `civicase` folder? If it is installed, you should get errors while commiting files. Please set it up in local, and fix all linting errors.